fourth_task/src/Shipment.ts

```diff
@@ -3,14 +3,32 @@ import { Letter } from './Letter';
 import { Package } from './Package';
 import { Oversized } from './Oversized';
 
+function format(_, $, propertyDesciptor: PropertyDescriptor): any {
+  propertyDesciptor.value = function (...args: any[]) {
+
+      return this.marks.reduce((acc, value) => {
+
+        if (value === 'Do Not Leave') {
+          value = `${value} if address not at home`;
+        }
+  
+        const message = `**MARK ${value.toUpperCase()} **`;
+  
+        return acc ? `${acc}\n ${message}` : message;
+      }, '')
+  }
+  return propertyDesciptor;
+};
+
 export interface ShipmentInterface {
   shipmentId: number,
   toAddress: string,
   fromAdress: string,
   toZipCode: string,
   fromZipCode: string,
   weight: number,
-  marks?: string[] 
+  marks?: string[],
+  marksMessages?: string | null
```

Comment:
@dianalemen, what purpose of adding this array into `ShipmentInterface`?

Reply:
@AlexHladin in order to use in onShop method

Reply:
@dianalemen, decorator only should know about marks. If your code will meet this criterion you will not need marks inside the interface.

Reply:
@AlexHladin I've got that this is unnecessary. Just answer because this is the question :)